lib/util/ios-proxy.js

```diff
@@ -14,6 +14,10 @@ const ROOT_URI = getPref("extensions." + id + ".sdk.rootURI");
 const TOOLS_URI = ROOT_URI + "tools/";
 
 let proxyProcess;
+let binaryName = "ios_webkit_debug_proxy";
+if (Runtime.OS == "WINNT") {
+  binaryName = "ios-webkit-debug-proxy.exe";
```

Comment:
That would require us to fork the project though, which is something that we haven't done for any other. I don't see much value in doing that at this point that would offset the maintenance costs.